events/wc_crisis_events.txt

```diff
@@ -701,6 +701,258 @@ narrative_event = {
 	}
 }
 
+### CRISIS: AWAKENING OF THE AMATHET ###
+# this crisis mostly serves as a counterbalance to the invading qiraji, and generally should fire not long after Uldum's isolation ends and it gets stomped but can happen anytime
+# Opening event: The Tombs Are Stirring
+# This serves as a pinger if an evil religion follower owns major holding
+character_event = {
+	id = WCCRS.200
+
+	is_triggered_only = yes
+	hide_window = yes
+
+	only_playable = yes
+
+	trigger = {
+		# Can't happen twice
+		NOT = {	has_global_flag = amathet_awakening_happened }
+		
+		# Uldum must be open to everyone
+		OR = {
+			has_game_rule = { name = uldum_isolation value = off }
+			has_global_flag = uldum_has_opened_flag
+		}
+
+		# Needs to control any of these important Tol'vir counties
+		OR = {
+			has_landed_title = c_orsis
+			has_landed_title = c_ramkahen
+			has_landed_title = c_neferset
+			has_landed_title = c_uldum
+			has_landed_title = c_tolnariv
+		}
+		
+		# Must be a public follower of an evil religion
+		evil_public_religion_trigger = yes
+	}
+
+	immediate = {
+		set_global_flag = amathet_awakening_happened	# Blocks the event from firing twice during one playthrough
+
+		# Check for an eligible Titanic/Suntouched tol'vir
+		any_playable_ruler = {
+			limit = {
+				OR = {
+					true_religion = mystery_of_the_makers
+					true_religion = suntouched
+				}
+				culture = tolvir
+			}
+			save_event_target_as = target_amathet_wakener
+		}
+		
+		# The following event is sent to a valid recipient that can go and explore the tombs
+		event_target:target_amathet_wakener = { narrative_event = { id = WCCRS.210 } }
+	}
+}
+
+### A Great Stir in the East
+# This is initial event for the Amathet Wakener
+narrative_event = {
+	id = WCCRS.210
+	title = EVTTITLE_WCCRS_210
+	desc = EVTDESC_WCCRS_210
+	picture = GFX_evt_uldum
+	border = GFX_event_narrative_frame_war
+	
+	is_triggered_only = yes
+
+	# Let's check this out
+	option = {
+		name = EVTOPTA_WCCRS_210
+
+		event_target:target_amathet_wakener = { character_event = { id = WCCRS.215 } }
+	}
+}
+
+### Tomb of the Sun King
+# Last chance to back out
+narrative_event = {
+	id = WCCRS.215
+	title = EVTTITLE_WCCRS_215
+	desc = EVTDESC_WCCRS_215
+	picture = GFX_evt_uldum
+	border = GFX_event_narrative_frame_war
+	
+	is_triggered_only = yes
+
+	# Delve deeper
+	option = {
+		name = EVTOPTA_WCCRS_215
+
+		event_target:target_amathet_wakener = { character_event = { id = WCCRS.220 } }
+
+		ai_chance = {
+			factor = 1
+		}
+	}
+
+	# This is a bad idea
+	option = {
+		name = EVTOPTB_WCCRS_215
+
+		custom_tooltip = {
+			text = stops_the_event_chain
```

Comment:
I'd change stops the event chain to something like 'You leave the tomb'. It kinda makes the outcome obvious as is, and breaks the implied fourth wall. Just a suggestion though.
```suggestion
			text = you_leave_the_tomb
```